api/src/main/java/marquez/service/models/ExtensibleJsonModel.java

```diff
@@ -0,0 +1,23 @@
+package marquez.service.models;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/** Base class for JSON models that supports additional properties for forward compatibility. */
+public class ExtensibleJsonModel {
```

Comment:
_Minor_: Mind if we use `BaseJsonObject` instead? just to keep aligned with the current naming `Base*`.

Reply:
I'm fine with that, but it may be worth pointing out that many of the JSON objects don't extend from this- i.e., all of the facets, as they are already extensible.

Reply:
Hmm, yeah fair point. And it's important to make that distinction. This is super nit-picky, so feel free to ignore, but  `ExtensibleJsonObject` just reads better 👌

Reply:
Ah! I already merged before I saw your last comment. `BaseJsonObject` is what got through 🤷🏽‍♂️